Makefile

```diff
@@ -251,6 +251,27 @@ ggml-opencl.o: ggml-opencl.cpp ggml-opencl.h
 	$(CXX) $(CXXFLAGS) -c $< -o $@
 endif # LLAMA_CLBLAST
 
+ifdef LLAMA_HIPBLAS
+	ROCM_PATH   ?= /opt/rocm
```

Comment:
ROCm path shouldn't be hardcoded to `/opt/rocm`. It's common to use the env var `ROCM_PATH`  (also `ROCM_HOME` is sometime used). `/opt/rocm` should only be a fallback.

Reply:
This is why it uses `?=` to set it unless it is set already. `/opt/rocm` seems to be [AMD's chosen path](https://rocm.docs.amd.com/en/latest/understand/file_reorg.html).

Reply:
You are correct, my mistake.